lib/version.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2015 The appc Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package docker2aci
+
+import "github.com/appc/spec/schema"
+
+var Version = "0.0.0+was-not-built-correctly"
```

Comment:
Talked about it with @iaguis OOB - it might be better to move these variables into docker2aci main package instead if we want to keep the way docker2aci is built.